### ISSUE TITLE
Setting weak passwords should be forbidden

### ISSUE THREAD

**Issue by jibel** [canonical/ubuntu-desktop-installer#1382]
Lunar 20230709

It is possible to set very weak passwords on the user setup page, like a single letter.
Ubuntu shouldn't contribute to making the ecosystem weaker than it already is and enforce minimal password strength. 
Libraries such as libpwquality or similar tools are useful to create stronger passwords.

**Comment by seb128** [comment]
The dialog currently provides feedback on the password strength but don't enforce it by preventing installing using a weak password. If we decide to be more restrictive we need design input on the minimal level to enforce (is a medium password like 'ImTryingUbuntu' ok or not?)

**Comment by dylanmtaylor** [comment]
I disagree with this. Perhaps an approach like Fedora where it says you have a weak password and makes you click twice to continue would be a good solution.
